README.md

```diff
@@ -59,7 +59,7 @@ The reload flag here will allow the service to restart if you make a change to t
 
 Ajv validator defaults to running on `http://localhost:5002`.
 
-You can override this by setting the `AJV_VALIDATOR_SCHEME`, `AJV_VALIDATOR_HOST`, and `AJV_VALIDATOR_PORT` environment variables.
+You can override this by setting the `AJV_VALIDATOR_SCHEME`, `AJV_VALIDATOR_HOST`, and `AJV_VALIDATOR_PORT` environment variables. These can be set by creating a `.env` file in the route of the repo file structure. There is a `.development.env` file with defaults that you can look at to see how they need to be set.
```

Comment:
I think we should leave the `.development.env` sentence if we are planning to add this on another ticket